rbc/heavydb/remoteheavydb.py

```diff
@@ -504,6 +504,36 @@ def thrift_client(self):
             self._thrift_client = self.client
         return self._thrift_client
 
+    def get_table_function_names(self, *, runtime_only=False):
```

Comment:
Personally not a fan of using `get/set` in function names. Does not look Pythonic.

Reply:
Do you have a better naming suggestions? I was using the same names as the thrift endpoint.

Reply:
I would just remove the prefix `get_`